fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/config/LdapConfig.java

```diff
@@ -55,10 +55,8 @@ SpringSecurityLdapTemplate springSecurityLdapTemplate() throws Exception {
     MessageFormat userDnPattern = new MessageFormat("uid={0},ou=users");
     String userSearchBase = "";
     String userSearchFilter;
+    String userIdAttribute;
     String groupSearchFilter = "(uniqueMember={0})";
     String groupRoleAttributes = "cn";
-    String groupUserAttributes = "";
```

Comment:
SO ... ONE thought, changing the attribute name in this way... vs deprecated handling of it.  NOMINALLY should have a better path to allow migration vs. a complete breaking change, while trying to keep some of the improved functionality under a different path.  Looking at some of this think this would definitely improve group handling but I think we need to provide a migration path.

Reply:
we add v2 provider